build-runtime.py

```diff
@@ -98,7 +99,7 @@ def install_sources (sourcelist):
 			p = subprocess.Popen(["dpkg-source", "-x", "--no-copy", dsc_file, os.path.join(dest_dir,ver)], stdout=subprocess.PIPE)
 			for line in iter(p.stdout.readline, ""):
 				if args.verbose or re.match('dpkg-source: warning: ',line):
-					print line,
+					print(line)
```

Comment:
This changes the meaning of the code by adding an additional newline. `line` already ends with a newline at this point.

```suggestion
					print(line, end='')
```